### ISSUE TITLE
Bootstrap dependency is pinned to an insecure version of bootstrap

### ISSUE THREAD

**Issue by cjcodes** [DesignRevision/shards-ui#55]
## Expected Behavior
`npm audit` should pass

## Current Behavior
`npm audit` shows a vulnerability in 4.1.3:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Cross-Site Scripting                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ bootstrap                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.4.1 <4.0.0 || >=4.3.1                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ shards-react                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ shards-react > shards-ui > bootstrap                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/891                             │
└───────────────┴──────────────────────────────────────────────────────────────┘

```

## Possible Solution
Upgrade bootstrap to >= 4.3.1

## Steps to Reproduce
1. Install `shards-react` or any other module that depends on this repo
2. Run `npm audit`

**Comment by cjcodes** [comment]
I would PR this, but I dunno how to manually verify that there are no regressions.

**Comment by cookieshell** [comment]
Fixed in #57
